use-cases/org-three-way-ecs.md

```diff
@@ -1,43 +1,49 @@
-# OrganizationSetup - Three way Cross-Account - ECS
+# Organizational Setup: Multiple Accounts and ECS
```

Comment:
ecs and k8s three-way titles are different.
i rather keep the "three way" naming, since customers and TAMs are used to it.